index.css

```diff
@@ -1,326 +1,562 @@
-    pointer-events: none;
-    opacity: 0;
+.homework-modal{
+  display: block;
+  margin-left: 30px;
+  color: rgba(0,0,0,.9);
+  font-size: 15px;
+  font-weight: 400;
 }
 
-.show{
-    pointer-events: unset;
-    opacity: 1;
+.close-container-homework{
+  position: absolute;
+  margin-left: 180px;
+  margin-top: -40px;
 }
 
-.homeworkMoldal{
-    display: block;
-    margin-left: 30px;
-    color: rgba(0,0,0,.9);
-    font-size: 15px;
-    font-weight: 400;
+.homework-item{
+  height: 32px;
+  display: grid;
+  align-items: center;
+  margin-top: 20px;
 }
 
-.close3{
-    position: absolute;
-    margin-left: 180px;
-    margin-top: -40px;
+.homework-item:hover{
+  background-color: rgba(163,172,182,.2);
 }
 
-.close3:hover{
-    color: rgba(68,136,255,1);
+.container-form{
+  display: grid;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  height: 250px;
+  width: 700px;
+  margin: auto;
+  box-shadow: 0 10px 20px rgba(163,172,182,0.5), 0 6px 6px rgba(163,172,182,0.5);
+  background-color: var(--color_1);
+  margin-top: -30px;
 }
 
-.work{
-    height: 32px;
-    display: grid;
-    align-items: center;
-    margin-top: 20px;
+.modal-create-task{
+  display: grid;
+  grid-template-rows: 40px 40px 40px 40px;
+  grid-column-gap: 20px;
 }
 
-.work:hover{
-    background-color: rgba(163,172,182,.2);
+.activity{
+  width: 90%;
+  height: 30px;
+  font-size: 20px;
+  font-family: inherit;
+  color: rgba(0,0,0,.9);
+  margin-left: 20px;
+  margin-top: 10px;
 }
 
-.containerForm{
-    display: grid;
-    z-index: 1;
-    top: 0;
-    left: 0;
-    bottom: 0;
-    right: 0;
-    height: 700px;
-    width: 800px;
-    margin: auto;
-    box-shadow: 0 10px 20px rgba(163,172,182,0.5), 0 6px 6px rgba(163,172,182,0.5);
-    pointer-events: none;
-    opacity: 0;
-    background-color: white;
-    margin-top: -30px;
-    margin-bottom: 30px;
+.label-form-1{
+  margin-top: 20px;
+  margin-left: 20px;
+  height: 30px;
 }
 
-.show2{
-    pointer-events: unset;
-    opacity: 1;
+.label-form-2{
+  margin-top: 10px;
+  margin-left: 20px;
+  height: 30px; 
 }
 
-.modalHome{
-    display: grid;
-    grid-template-columns: 1fr 30px;
-    grid-column-gap: 20px;
+.activity:hover{
+  border-bottom: 1px solid black;
 }
 
-textarea{
-    width: 90%;
-   height: 30px;
-   border: none;
-   font-size: 20px;
-   font-family: inherit;
-   color: rgba(0,0,0,.9);
-   margin-left: 20px;
-   margin-top: 10px;
+a{
+  text-decoration: none;
 }
 
-textarea:hover{
-    border-bottom: 1px solid black;
+.close-containerForm{
+  font-size: 25px;
+  color: black;
+  font-weight: normal;
+  position: absolute;
+  margin-left: 670px;
 }
 
-a{
-    text-decoration: none;
+.send-form-button{
+  position: absolute;
+  height: 30px;
+  width: 60px;
+  left: 1000px;
+  top: 230px;
 }
 
-.close{
-    font-size: 25px;
-    color: black;
-    font-weight: normal;
+.seeSections{
+  z-index: 1;
+  display: grid;
```

Comment:
remove display: grid please is not needed 
![image](https://user-images.githubusercontent.com/7244087/177218344-87b292b4-3657-47d0-8e00-502448396353.png)